doc/userguide/output/eve/eve-json-output.rst

```diff
@@ -107,13 +107,27 @@ YAML::
             extended: yes     # enable this for extended logging information
             # custom allows to control which tls fields that are included
             # in eve-log
-            #custom: [subject, issuer, serial, fingerprint, sni, version, not_before, not_after, certificate, chain]
+            #custom: [subject, issuer, serial, fingerprint, sni, version, not_before, not_after, certificate, chain, ja3_hash, ja3_str]
 
 The default is to log certificate subject and issuer. If ``extended`` is
 enabled, then the log gets more verbose.
 
 By using ``custom`` it is possible to select which TLS fields to log.
 
+JA3
+~~~
+
+JA3 fingerprints are logged one record per TLS session.
```

Comment:
on record? or did I misread this sentence?

Reply:
I was thinking it as one JSON record per TLS session. Should it be changed to something else?